Sources/PointFree/AtomFeed.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+import Html
+import HttpPipeline
+import MediaType
+import Prelude
+
+let atomFeedResponse =
+  writeStatus(.ok)
+    >-> respond(pointFreeFeed, contentType: .html)
+
+let pointFreeFeed = View<[Episode]> { episodes in
+  atomLayout.view(
+    AtomFeed(
+      author: AtomAuthor(
+        email: "support@pointfree.co",
+        name: "Point-Free"
+      ),
+      entries: episodes.map(atomEntry(for:)),
+      atomUrl: url(to: .feed(.atom)),
+      siteUrl: "/", // FIXME url(to: .root),
```

Comment:
can't we make this `.secretHome` or `.home(signedUpSuccessfully: nil)` right now?

Reply:
Yep yep.